### DISCUSSION TITLE
Performance: SimpleLogistics.FixedUpdate can be expensive

### DISCUSSION THREAD

**Discussion by JonnyOThan** [zer0Kerbal/SimpleLogistics#64] [General]
I noticed this when looking through someone's profile capture:
![image](https://github.com/zer0Kerbal/SimpleLogistics/assets/9804078/7b0ff0db-dec2-45d9-9c8f-ee5f96a60dd1)

[a comment by an automated account is not shown]

**Comment by zer0Kerbal** [comment]
@JonnyOThan Thank you. Not enough information to do more than keep it in mind.

@LisiasT

**Reply by JonnyOThan** [reply]
Um, there's a huge amount of obviously slow code in there.
